.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           command: "cargo build"
       - run:
           name: Run Tests (including doc tests)
-          command: "cargo test -- --exact --show-output"
```

Comment:
That should not be required. I believe that the changes in my PR fix this (some tests incorrectly ran in parallel).